com.github.Murmele.scram.yml

```diff
@@ -0,0 +1,31 @@
+app-id: com.github.Murmele.scram
+runtime: org.kde.Sdk
+runtime-version: 5.15-21.08
+sdk: org.kde.Sdk
+command: scram-gui
+rename-icon: scram
+rename-desktop-file: scram-gui.desktop
+rename-appdata-file: scram-gui.appdata.xml
+finish-args:
+- --socket=wayland
+- --socket=fallback-x11
+- --filesystem=home
+- --share=ipc
+modules:
+- boost.json
```

Comment:
having the boost module inline here is better

Reply:
done